panther_gpiod/include/panther_gpiod/gpio_driver.hpp

```diff
@@ -245,9 +254,10 @@ class GPIODriver
     {GPIOPin::GPIN2, "GPIN2"},
     {GPIOPin::LED_SBC_SEL, "LED_SBC_SEL"},
     {GPIOPin::SHDN_INIT, "SHDN_INIT"},
+    {GPIOPin::STAGE2_INPUT, "STAGE2_INPUT"},
     {GPIOPin::VDIG_OFF, "VDIG_OFF"},
     {GPIOPin::VMOT_ON, "VMOT_ON"},
-  };
+    {GPIOPin::MOTOR_ON, "MOTOR_ON"}};
```

Comment:
is it formatted correctly? shouldn't `};` be in the next line?

Reply:
Depends in there is `,` after last element. I copied it as it was in prealpha but it make sense to change that